eng/pipelines/coreclr/templates/helix-queues-setup.yml

```diff
@@ -7,6 +7,7 @@ parameters:
   container: ''
   pool: ''
   platform: ''
+  runtimeFlavorDisplayName: ''
```

Comment:
Please include this change in `libraries/helix-queues-setup.yml` as well just in case we need to use this via that template expansion ever so that we don't need to investigate why it is not setting it correctly.